### PR TITLE
[env/github] Make profile urls clickable

### PR DESCRIPTION
**Pull request by malaikaaiyar**
fixes https://github.com/sparkletown/internal-sparkle-issues/issues/872

### CODE REVIEW

File: src/components/organisms/ProfileModal/UserInformationContent/UserInformationContent.tsx [hunk 1]
```diff
@@ -121,8 +122,10 @@ export const UserInformationContent: React.FunctionComponent<UserInformationCont
         .map((question) => (
           <div key={question.name} className="question-section">
             <div className="question">{question.text}</div>
-            {/* @ts-ignore question.name is a correct index for type User */}
-            <div>{user?.[question.name]}</div>
+            <div>
```
Comment (denisdimitrov) [inline src/components/organisms/ProfileModal/UserInformationContent/UserInformationContent.tsx:125]: Do you need this `div` now that you have the component `<RenderMarkdown/>` ?

Comment (malaikaaiyar) [reply src/components/organisms/ProfileModal/UserInformationContent/UserInformationContent.tsx:125]: Probably not, but let me see if it looks different

Comment (malaikaaiyar) [reply src/components/organisms/ProfileModal/UserInformationContent/UserInformationContent.tsx:125]: Yeah- it looks the same so I'll delete it

**Review by denisdimitrov — APPROVED** [review]
LGTM!

**Pull request merged**